server/db.js

```diff
@@ -10,5 +10,14 @@ const connectDB = async () => {
 
   console.log(`MongoDB Connected: ${conn.connection.host}`.cyan.underline.bold);
 };
+const Schema  = mongoose.Schema;
+const ProfileSchema =  new Schema({ 
+		FirstName: String,
+		LastName: String,
+		Description: String,
+		Availability: 	Date
```

Comment:
Are we planning to have a date range (start-end) or availability throughout the (Mon-Fri) week? :)
(a single date won't fit our use case for availability)